samples/apps/copilot-chat-app/webapi/Options/PromptsOptions.cs

```diff
@@ -61,89 +54,83 @@ public PromptSettings(PromptsConfig promptsConfig)
     /// </summary>
     internal double DocumentMemoryMinRelevance { get; } = 0.8;
 
-    internal string KnowledgeCutoffDate => this._promptsConfig.KnowledgeCutoffDate;
-    internal string InitialBotMessage => this._promptsConfig.InitialBotMessage;
-
-    // System commands
-    internal string SystemDescriptionPrompt => this._promptsConfig.SystemDescription;
-    internal string SystemResponsePrompt => this._promptsConfig.SystemResponse;
-
-    // Intent extraction commands
-    internal string SystemIntentPrompt => this._promptsConfig.SystemIntent;
-    internal string SystemIntentContinuationPrompt => this._promptsConfig.SystemIntentContinuation;
-
+    // System
+    [Required, NotEmptyOrWhitespace] public string KnowledgeCutoffDate { get; set; } = string.Empty;
+    [Required, NotEmptyOrWhitespace] public string InitialBotMessage { get; set; } = string.Empty;
+    [Required, NotEmptyOrWhitespace] public string SystemDescription { get; set; } = string.Empty;
+    [Required, NotEmptyOrWhitespace] public string SystemResponse { get; set; } = string.Empty;
     internal string[] SystemIntentPromptComponents => new string[]
     {
-        this.SystemDescriptionPrompt,
-        this.SystemIntentPrompt,
+        this.SystemDescription,
+        this.SystemIntent,
         "{{ChatSkill.ExtractChatHistory}}",
-        this.SystemIntentContinuationPrompt
+        this.SystemIntentContinuation
     };
+    internal string SystemIntentExtraction => string.Join("\n", this.SystemIntentPromptComponents);
 
-    internal string SystemIntentExtractionPrompt => string.Join("\n", this.SystemIntentPromptComponents);
+    // Intent extraction
+    [Required, NotEmptyOrWhitespace] public string SystemIntent { get; set; } = string.Empty;
+    [Required, NotEmptyOrWhitespace] public string SystemIntentContinuation { get; set; } = string.Empty;
 
-    // Memory extraction commands
-    internal string SystemCognitivePrompt => this._promptsConfig.SystemCognitive;
-    internal string MemoryFormat => this._promptsConfig.MemoryFormat;
-    internal string MemoryAntiHallucination => this._promptsConfig.MemoryAntiHallucination;
-    internal string MemoryContinuationPrompt => this._promptsConfig.MemoryContinuation;
+    // Memory extraction
+    [Required, NotEmptyOrWhitespace] public string SystemCognitive { get; set; } = string.Empty;
```

Comment:
`SystemCognitive` -> `CognitiveMemoryDescription`

Reply:
Let's do a separate pass for changing these values.